precommit/src/main/shell/test-patch.d/gradle.sh

```diff
@@ -23,65 +23,47 @@ declare -a GRADLE_ARGS=()
 
 function gradle_usage
 {
-  yetus_add_option "--gradle-cmd=<cmd>" "The 'gradle' command to use (default 'gradle')"
   yetus_add_option "--gradlew-cmd=<cmd>" "The 'gradlew' command to use (default 'basedir/gradlew')"
 }
 
 function gradle_parse_args
 {
-  local i
-
-  for i in "$@"; do
-    case ${i} in
-      --gradle-cmd=*)
-        delete_parameter "${i}"
-        GRADLE=${i#*=}
-      ;;
-      --gradlew-cmd=*)
```

Comment:
usage above still claims we can pass a gradlew comand in. either remove it or add the handling back in here.

Reply:
Doh! Yeah that's clearly wrong. Thanks!